src/osdag/design_type/compression_member/Column.py

```diff
@@ -1287,22 +1298,12 @@ def save_design(self, popup_summary):
             if self.column_properties.flange_slope != 90:
                 select_section_img = "Slope_Beam"
             else:
-                select_section_img = "Parallel_Beam"
```

Comment:
Is this comment needed?

Reply:
For the CHS, RHS and SHS section profile it might/should come in use and since it was part of the original code, I commented it rather than deleting it altogether.

Reply:
OK. Ideally add a comment for why it's needed the next time.